osh/client/osh-cli

```diff
@@ -79,6 +79,9 @@ def main():
     except kobo.exceptions.ImproperlyConfigured as ex:
         print(f"\n\nError: Improperly configured: {ex}", file=sys.stderr)
         sys.exit(3)
+    except Exception as ex:  # noqa: B902
+        print(f"\n\nError: {ex}", file=sys.stderr)
+        sys.exit(1)
```

Comment:
I can understand that OSError may contain non-network exceptions, but is there a reason not to be consistent with the original exceptions in `kobo`? i.e., the `http.client.HTTPException`, `TimeoutError` were added here. Why not use:

    except (socket.herror, socket.gaierror, socket.timeout, httplib.CannotSendRequest, xmlrpclib.ProtocolError)

so that it's consistent with exceptions kobo raises. @lzaoral

Reply:
Sure, why not.  I just though that would want to catch all exceptions related to network issues.

Reply:
Thanks, updated.

Reply:
Thank you!  I did not included that in the suggestion above but you also have to add respective imports,  Also `httplib` and `xmlrpclib` do not exist in Python 3, they are just Python 2 compatibility aliases used internally by `kobo`.

Reply:
Hmm, even with the correct imports, the code is not syntactically correct:
```python
Traceback (most recent call last):
  File "/github/home/.cache/pre-commit/repobn0nuff3/py_env-python3/lib/python3.12/site-packages/pre_commit_hooks/debug_statement_hook.py", line 57, in check_file
    ast_obj = ast.parse(f.read(), filename=filename)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib64/python3.12/ast.py", line 52, in parse
    return compile(source, filename, mode, flags,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "osh/client/osh-cli", line 82
    except (socket.herror, socket.gaierror, socket.timeout, httplib.CannotSendRequest, xmlrpclib.ProtocolError) as ex
                                                                                                                     ^
SyntaxError: expected ':'
```

Reply:
ops I made silly mistakes. I now get the point why you tweaked the exception in original thread!